main/src/library/Ipv6Addr.flix

```diff
@@ -30,33 +30,124 @@ instance FromString[Ipv6Addr] {
     pub def fromString(x: String): Option[Ipv6Addr] = Ipv6Addr.fromString(x)
 }
 
+instance ToString[Ipv6Addr] {
+    pub def toString(x: Ipv6Addr): String = Ipv6Addr.toString(x)
+}
+
 mod Ipv6Addr {
+    import java.lang.Integer
+    import java.lang.NumberFormatException
     import java.net.InetAddress
     import java.net.UnknownHostException
 
     use Ipv6Addr.Ipv6Addr
 
+    def toUnsigned8(x: Int8): Int32 = Int32.bitwiseAnd(Int8.toInt32(x), 0xFF)
```

Comment:
I would move the private helper functions last in the module, so the public functions come first.

Reply:
We also need to document each function